### ISSUE TITLE
Replace tangential forces to displacement (ex28)

### ISSUE THREAD

**Issue by CODER591** [mfem/mfem#2247]
Hello,
I am trying to model an elasticity problem on 2D mesh, in 2D space, and I am considering rework of example 28.
The only thing I need to do is to apply forces to one side of that figure (as in the plane stress ) and get the displacement in 2d space
Image:
![Screenshot from 2021-05-17 18-21-10](https://user-images.githubusercontent.com/33939665/118513919-c969b180-b73c-11eb-8a63-cec63dec0ce6.png)

However, I have no idea how to specify that I want the displacement, not tangential force, to get surface change only in 2D
How should I change the code?
I have spent a lot of time, playing with a lot of parameters, and performing attempts to rework it by myself, but unsuccessfully.
Generally, I have tried to remove constraints, but what should I use instead of `SchurConstrainedSolver`?

```
Array <int> constraint_atts(2);   
constraint_atts[0] = 1;  // attribute 1 bottom   
constraint_atts[1] = 4;  // attribute 4 left side   
Array<int> lagrange_rowstarts;   
SparseMatrix* local_constraints =   BuildNormalConstraints(*fespace, constraint_atts, lagrange_rowstarts);
GSSmoother M(A);   
SchurConstrainedSolver * solver =      new SchurConstrainedSolver(A, *local_constraints, M);   solver->SetRelTol(1e-5); 
```
Thanks in advance!

**Comment by tzanio** [comment]
Is the left wall vertical as in the image or slanted as in example 28?

If it is vertical, it is better to just use a simplified version of Example 2, where you have only one material and the force vector on the right boundary is in `x` instead of `y` direction.

**Comment by CODER591** [comment]
Thanks for the answer! 
Finally, I have reworked the ex2 and, graphically I achieved a similar result as in the ex28.
But, how and where in code I should specify `x` direction of force vector? I totally don't get it.
As from outputs, it looks like there is `z`-direction always changing.
Should I operate with linear algebra on force vector, to change its direction? From vertical `Z` to horizontal `X`?
Also, vector size == 4, how should its values be interpreted?` (x,y,z,?)`
![Screenshot from 2021-05-18 23-33-48](https://user-images.githubusercontent.com/33939665/118719749-863d3a80-b831-11eb-8bbd-dbddc9b9972f.png)
![Screenshot from 2021-05-18 23-37-42](https://user-images.githubusercontent.com/33939665/118720268-17141600-b832-11eb-948c-bfb6c9414a0f.png)

[ex2_reworked_in_2d.txt](https://github.com/mfem/mfem/files/6503729/ex2_reworked_in_2d.txt)
